core/data/src/test/java/uk/gov/gchq/gaffer/data/elementdefinition/view/ViewTest.java

```diff
@@ -681,6 +682,212 @@ public void shouldReturnFalseWhenViewHasEmptyPostTransformEdgeFilters() {
         assertFalse(result);
     }
 
+    @Test
+    public void shouldAddGlobalPropertiesToGroup() {
+        // Given
+        final View view = new View.Builder()
+                .globalEntities(new GlobalViewElementDefinition.Builder()
+                        .groups(TestGroups.ENTITY)
+                        .properties(TestPropertyNames.PROP_1, TestPropertyNames.PROP_2)
+                        .build())
+                .entity(TestGroups.ENTITY)
+                .build();
+
+        // When
+        view.expandGlobalDefinitions();
+
+        // Then
+        assertEquals(Sets.newHashSet(TestPropertyNames.PROP_1, TestPropertyNames.PROP_2),
+                Sets.newHashSet(view.getEntity(TestGroups.ENTITY).getProperties()));
+    }
+
+    @Test
+    public void shouldIgnoreEmptyGlobalProperties() {
```

Comment:
It doesn't look like they have been 'ignored', the properties has been set to an empty set. Perhaps rename the test method.